iOS/Flows/Conversation/ConversationViewController.swift

```diff
@@ -214,7 +214,7 @@ class ConversationViewController: FullScreenViewController,
         let messages = controller.messages
         var snapshot = self.dataSource.snapshot()
 
-        let section = ConversationSection(cid: controller.cid)
+        let section = ConversationSection(sectionID: controller.cid!.description)
```

Comment:
The section is no longer always a conversation. Now it could be a collection of channels so I made the id name more generic.